application-forum-ui/src/main/resources/ForumCode/Translations.xml

```diff
@@ -295,7 +295,8 @@ templateProvider.forum.name=Forum
 templateProvider.forum.description=Start a conversation
 
 ## Forum Administration Section
-forum.administration.configuration.title=Configuration</content>
+forum.administration.configuration.title=Configuration
+forum.administration.warningMessage=It seems like you're trying to access this instance locally.\nThis functionally works only for valid (extern) hosts.</content>
```

Comment:
``\n`` won't have the expected effect when the message is injected in HTML so I'd rather not use it (you make the assumption that the message is used only from wiki syntax where it generates a BR).

Also, I think the message can be improved.

```suggestion
forum.administration.warningMessage=It seems like you're accessing this instance through localhost. This will prevent you from sharing URLs on social media.</content>
```